packages/amesos2/src/Amesos2_Superludist.cpp

```diff
@@ -58,60 +58,36 @@ namespace Amesos2 {
 #endif
 
 #ifdef HAVE_TPETRA_INST_INT_INT
```

Comment:
At some point, Amesos2 needs to revise how it does ETI.  Right now, it's super brittle.